composer.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "roave/you-are-using-it-wrong",
+    "description": "Composer plugin enforcing strict type checks in downstream package dependants",
+    "type": "composer-plugin",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marco Pivetta",
+            "email": "ocramius@gmail.com"
+        }
+    ],
+    "require": {
+        "php":                       "^7.3.0",
+        "ext-json":                  "*",
+        "composer-plugin-api":       "^1.0.0",
+        "ocramius/package-versions": "^1.4.0",
+        "vimeo/psalm":               "dev-master"
```

Comment:
According to @muglug, this needs another release, happening in the next few days

Reply:
Requiring Psalm here of course could cause a possible type-conflict with any project that requires a different version of the [libraries required by psalm](https://github.com/vimeo/psalm/blob/master/composer.json#L12). I wonder if instead of doing that it would be possible to download a stand-alone Psalm as necessary, either as .phar or as a separate install with all its own dependencies. A bit like how [Symfony's PHPUnit bridge downloads PHPUnit to a .phpunit directory](https://github.com/symfony/phpunit-bridge/blob/master/bin/simple-phpunit#L75).

Reply:
This is feasible, although out of scope for this specific PR, due to it already being quite large.

Could you open a new issue about it? Downloading the `.phar` (or similar) would probably work well, but I'd generally prefer having a version conflict over having to design a custom installer and dependency tracking mechanism (composer is already extremely good at this, which is why I don't use standalone `.phar` anymore)

Reply:
https://github.com/Roave/you-are-using-it-wrong/issues/2